src/healthkit.cs

```diff
@@ -17,6 +17,13 @@
 using System.ComponentModel;
 using CoreLocation;
 using UniformTypeIdentifiers;
+#if MONOMAC
+using AppKit;
+using UIViewController = AppKit.NSViewController;
+#else
+using UIKit;
+using NSViewController = Foundation.NSObject;
```

Comment:
this isn't really necessary in the context of this PR but thought it would be good to include to maintain consistency b/w platforms if there is a UI/NSViewController use case in the future..